src/components/AddItems.js

```diff
@@ -7,9 +8,15 @@ const db = firebase.firestore().collection('shopping_list');
 const arrayUnion = firebase.firestore.FieldValue.arrayUnion;
 
 const AddItemsToList = () => {
+  const userToken = localStorage.getItem('token');
   const [shoppingListItemName, setShoppingListItemName] = useState('');
   const [daysLeftForNextPurchase, setDaysLeftForNextPurchase] = useState(7);
 
+  const [shoppingList, loading, error] = useCollectionData(
```

Comment:
Yes, this is great! I'm actually getting clearer on Firebase hooks by following your process, team! So good.